Content.Shared/Speech/Components/VulgarAccentComponent.cs

```diff
@@ -0,0 +1,34 @@
+using Robust.Shared.GameStates;
+
+namespace Content.Shared.Speech.Components;
+
+[RegisterComponent, NetworkedComponent]
+public sealed partial class VulgarAccentComponent : Component
+{
+    [DataField]
+    public string[] SwearWords =
+    {
+        "FUCK",
+        "FUCKING HELL",
+        "GOD DAMN",
+        "SHIT",
+        "BIG SACK OF SHIT",
+        "SON OF A BITCH",
+        "BITCH",
+        "COCK",
+        "COCKSUCKER",
+        "BONER SHIT COCKS",
+        "MOTHERFUCKER",
+        "YOU BASTARD",
+        "DICK",
+        "DICKBAG",
+        "ASSHOLE",
+        "SHITTY ASS FUCKHOLE",
+        "GOOD HEAVENS",
+        "SON OF A CLUWNE"
+    };
```

Comment:
well it's still bad, because it's still in code
i suggest you to use LocalizedDataset
i think #30455 will help you a little, i used quite similar thing there

Reply:
Whoa thank you that looks way more efficient give me a few minutes to make a fix.